File: doc-examples/hibernate-reactive-example-groovy/src/test/groovy/example/PostgresHibernateReactiveProperties.java
package example;

import io.micronaut.test.support.TestPropertyProvider;

import java.util.Map;

public interface PostgresHibernateReactiveProperties extends TestPropertyProvider {

    @Override
    default Map<String, String> getProperties() {
        return Map.of(
                "jpa.default.properties.hibernate.hbm2ddl.auto", "create-drop",
                "jpa.default.reactive", "true",
                "jpa.default.properties.hibernate.connection.db-type", "postgres"
        );
    }
}

File: doc-examples/hibernate-reactive-example-kotlin/src/test/kotlin/example/PostgresHibernateReactiveProperties.kt
package example

import io.micronaut.test.support.TestPropertyProvider

interface PostgresHibernateReactiveProperties : TestPropertyProvider {

    override fun getProperties(): Map<String, String> {
        return java.util.Map.of(
                "jpa.default.properties.hibernate.hbm2ddl.auto", "create-drop",
                "jpa.default.reactive", "true",
                "jpa.default.properties.hibernate.connection.db-type", "postgres"
        )
    }


}
